core/gui/src/app/dashboard/service/admin/user/admin-user.service.ts

```diff
@@ -30,12 +32,21 @@ export class AdminUserService {
   }
 
   public updateUser(uid: number, name: string, email: string, role: Role): Observable<void> {
-    return this.http.put<void>(`${USER_UPDATE_URL}`, {
-      uid: uid,
-      name: name,
-      email: email,
-      role: role,
-    });
+    return this.http
+      .put<void>(`${USER_UPDATE_URL}`, {
+        uid: uid,
+        name: name,
+        email: email,
+        role: role,
+      })
+      .pipe(
+        catchError((error: unknown) => {
```

Comment:
Why do you need to catch error here and throw error again? I believe this step is unnecessary. You can check the other error reporting on frontend. Usually we just need to throw error on backend, and show error on component's subscribe.

Reply:
Resolved!